catalog/index.html

```diff
@@ -1,37 +1,97 @@
-<html>
-  <head>
-    <title>
-      Manubot - Docs
-    </title>
-    <link
-      href="https://fonts.googleapis.com/css?family=Quicksand"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css?family=Inconsolata"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
-    />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" type="text/css" href="../styles.css" />
-    <link rel="stylesheet" type="text/css" href="../coming-soon.css" />
-  </head>
-  <body>
+---
+layout: default
+title: Manubot Catalog
+logo: catalog-logo.svg
+header: Manubot
+subheader: CATALOG
+---
+
+<section>
+  <div class="row">
     <div>
-      <h2>Coming Soon</h2>
-      <i class="fas fa-drafting-compass fa-lg"></i><br>
-      <br>
-      <div>
-        We are still in the process of gathering and presenting a complete list of papers written with Manubot.
-        In the meantime, please take a look at the <a href="..#made-with-manubot">list of notable Manubot papers</a> on the main page.
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+      est laborum.
```

Comment:
Here is some potential text for the page:

Browse manuscripts that were made with Manubot.
Manuscripts are sorted by date last modified.
Note that many manuscripts may still be in-progress.

Don't see your manuscript here? Or is a manuscript missing a preprint or journal link? Update the catalog by pull request in [this repository](https://github.com/manubot/catalog).

Reply:
![image](https://user-images.githubusercontent.com/8326331/60461957-11998c80-9c16-11e9-9514-b9436e55ef08.png)

Reply:
The catalog is not just for "favorites". It should be inclusive of all manuscripts as long as they have some minimum level of original content (and have manubot html and source code available).